lib/inkdrop-note-templates.js

```diff
@@ -4,17 +4,28 @@ const { Liquid } = require('liquidjs');
 const setupTemplates = async () => {
     const db = inkdrop.main.dataStore.getLocalDB()
     const TEMPLATE_NOTEBOOK_NAME = "_Templates";
-    const templateDocs = await db.books.findWithName(TEMPLATE_NOTEBOOK_NAME);
-    if (!templateDocs) {
-        console.log(`Not found ${TEMPLATE_NOTEBOOK_NAME} notebook`)
-        return [];
+    const TEMPLATE_TAG_NAME = "Template";
+
+    let templateNotes = Array();
```

Comment:
```suggestion
    let templateNotes = [];
```

We can use array literal instead of it.

Reply:
Okay! Let's use array literal instead!